opba-banking-rest-impl/src/test/java/de/adorsys/opba/tppbanking/impl/controller/TestTppBankSearchController.java

```diff
@@ -0,0 +1,54 @@
+package de.adorsys.opba.tppbanking.impl.controller;
+
+import de.adorsys.opba.tppbanking.impl.BaseMockitoTest;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import static de.adorsys.opba.tppbanking.impl.TestProfiles.ONE_TIME_POSTGRES_RAMFS;
+
+@SpringBootTest
+@ActiveProfiles(ONE_TIME_POSTGRES_RAMFS)
+@AutoConfigureMockMvc
+class TestTppBankSearchController extends BaseMockitoTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    void testBankSearch() throws Exception {
+        mockMvc.perform(
+                MockMvcRequestBuilders.get("/v1/banking/search/bank-search")
+                        .header("Authorization", "123")
+                        .header("X-Request-ID", "01f4ec8e-8fb8-4e37-8912-bae6ff227231")
+                        .param("keyword", "commerz")
+                        .param("max", "10")
+                        .param("start", "0"))
+                .andExpect(MockMvcResultMatchers.status().isOk())
```

Comment:
Better to import MockMvcResultMatchers/MockMvcRequestBuilders statically

Reply:
ok